primo/data_parser/default_data.py

```diff
@@ -297,33 +297,45 @@ class _SupportedContent:
     ),
 }
 
+# note the names of these metrics must be identical to
+# a property of the Optimal Project class for the
+# efficiency calculation
 SUPP_EFF_METRICS = {
     "num_wells": _SupportedContent(
-        name="num_wells",
-        full_name="Number of Wells",
+        name="num_wells", full_name="Number of Wells", required_data="well_id"
     ),
     "num_unique_owners": _SupportedContent(
         name="num_unique_owners",
         full_name="Number of Unique Owners",
+        required_data="operator_name",
+        has_inverse_priority=True,
     ),
-    "dist_centroid": _SupportedContent(
-        name="dist_centroid",
-        full_name="Distance to Centroid [miles]",
-    ),
-    "elevation_delta": _SupportedContent(
-        name="elevation_delta",
+    "avg_elevation_delta": _SupportedContent(
+        name="avg_elevation_delta",
         full_name="Average Elevation Delta [m]",
+        required_data="elevation_delta",
+        has_inverse_priority=True,
     ),
     "age_range": _SupportedContent(
         name="age_range",
         full_name="Age Range [Years]",
+        required_data="age",
+        has_inverse_priority=True,
     ),
     "depth_range": _SupportedContent(
         name="depth_range",
         full_name="Depth Range [ft]",
+        required_data="depth",
+        has_inverse_priority=True,
     ),
     "record_completeness": _SupportedContent(
         name="record_completeness",
         full_name="Record Completeness",
     ),
+    "avg_dist_to_road": _SupportedContent(
+        name="avg_dist_to_road",
+        full_name="Distance to Road [miles]",
+        required_data="dist_to_road",
+        has_inverse_priority=True,
+    ),
```

Comment:
Could you please add `population_density` to the supported content. It has inverse priority (higher the population density, lower the efficiency). For the fill_missing_value attribute, set the default value as 0 and the domain as NonNegativeFloat. Also, please add an attribute for this column in the `WellDataColumnNames` class.

Reply:
This will happen in a later PR.